packages/office-ui-fabric-react/src/components/SwatchColorPicker/SwatchColorPicker.Props.ts

```diff
@@ -47,13 +47,13 @@ export interface ISwatchColorPickerProps {
    * Callback issued when the user hovers over a color cell.
    * Note, if no id or color is given, cells are not longer being hovered
    */
-  onCellHovered?: (id?: string, color?: string) => void;
+  onCellHovered?: (item?: IColorCellProps) => void;
```

Comment:
These are breaking changes which we try to avoid. If we're going to make many breaking changes to this, I think we should remove it from the docs page so that people don't take a dependency on it only to get a bunch of build breaks every time they update!